modules/installation-special-config-kmod.adoc

```diff
@@ -453,23 +378,39 @@ $ cd ../kvc-simple-kmod
 $ make install DESTDIR=${FAKEROOT}/usr/local CONFDIR=${FAKEROOT}/etc/
 ----
 
-. Get a tool called `filetranspiler` and dependent software:
+. Clone the fakeroot directory, replacing any symbolic links with copies of their targets:
```

Comment:
Showing my ignorance here but is this saying that the user should replace sym links with their targets, as opposed to the command actually doing the replacing? If so, maybe you could say, "Clone the fakeroot directory, making sure to replace any symbolic links with copies of their targets:"?

Reply:
No, the command is doing it.  Do you think the wording is sufficiently clear in that case?

Reply:
Ah, cool. In that case, it's likely clear enough to people who are doing this then. But you could edit slightly to something like, "Clone the fakeroot directory, replacing any symbolic links with copies of their targets, by running the following command:" 🤷